src/main/java/beans/UserBean.java

```diff
@@ -109,7 +104,7 @@ public void setPassword(String password) {
 		this.password = password;
 	}
 	
-	public boolean getGender() {
+	public boolean isGender() {
```

Comment:
it should probably renamed to something like is male or is female as isGender does not actually tell what is the value of this boolean, if the gender is true you still have no idea what gender it actually is.